test/functional/pdo_sqlsrv/skipif_server_old.inc

```diff
@@ -0,0 +1,19 @@
+<?php
+
+if (!extension_loaded("pdo") || !extension_loaded('pdo_sqlsrv'))
+    die("PDO driver cannot be loaded; skipping test.\n");
+
+require_once( "MsSetup.inc" );
+$conn = new PDO("sqlsrv:server = $server;", $uid, $pwd );
+if( ! $conn )
+{
+    die( "skip - could not connect during SKIPIF." );
+}
```

Comment:
I got the idea from skipif_protocol_not_tcp.inc :)

Reply:
hmmmm I agree with David too. If somehow it can't connect, the test should fail instead of being skipped. Should fix this in skipif_protocol_not_tcp.inc too (but maybe in a separate PR to the dev branch)